### PR TITLE
alpaca::cryptography

### PR DESCRIPTION
**Pull request by BlueCocoa**
RSA encode and decode function.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Llama-Group/Project-Llama/pull/12" target="_blank">ReviewNinja

File: llama_cpp/Cryptography/Cryptography.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//
+//  Cryptography.h
+//  Alpaca
+//
+//  Created by BlueCocoa on 2016/7/22.
+//  Copyright © 2016 BlueCocoa. All rights reserved.
```
Comment (0xsuu) [inline llama_cpp/Cryptography/Cryptography.h]: Change copy right. Maybe Apache License?

Comment (BlueCocoa) [inline llama_cpp/Cryptography/Cryptography.h]: Okay, changed to MIT license.

Comment (BlueCocoa) [inline llama_cpp/Cryptography/Cryptography.h]: !completed

**Comment by 0xsuu** [comment]
discussion related to license has been moved to #13

**Comment by 0xsuu** [comment]
@BlueCocoa Review with ninja review, add a star!

**Pull request merged**